docs/internals/contributing/writing-documentation.txt

```diff
@@ -420,3 +420,20 @@ Translating documentation
 
 See :ref:`Localizing the Django documentation <translating-documentation>` if
 you'd like to help translate the documentation into another language.
+
+.. _django-admin-manpage:
+
+Django-admin man page
+---------------------
+
+Sphinx can generate a manual page for the
+:doc:`django-admin </ref/django-admin>` command. This is configured in
+``conf.py``. Unlike other documentation output, this man page should be
+included in the Django repository and the releases in
+``docs/man/django-admin.1``.
+
+To generate an updated version of the man page, run ``make man`` in the
+``docs`` directory. The new man page will be written in
+``_build/man/django-admin.1``. Note that Sphinx will also generate another
+man page named ``django.1`` which contains the entire documentation. That
```

Comment:
Personally, I can't imagine anyone wanting to read 4MB of manpage to get to the Django docs. This seems to have been added in fad4a93275a6a5267912c9fc88cdeee44dd5572f, but not purposely for manpages.